src/components/Tour.vue

```diff
@@ -44,11 +44,45 @@ export default {
 				},
 				'ide-workspace': {
 					title: 'User Workspace',
-					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services and files.'
+					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services, custom processes and files.'
 				},
 				'ide-viewer': {
 					title: 'Data Viewer',
-					content: 'This area you can use to preview collections or inspect the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+					content: 'This area is for previewing collections or inspecting the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+				},
+
+				'connect-url': {
+					title: 'URL input',
+					content: `Paste the URL of the server you wish to connect to into this field. If you don't know any URL, you may head over to <a href="https://hub.openeo.org/">openEO Hub</a> to discover available servers.`
+				},
+				'connect-history': {
+					title: 'History of URLs',
+					content: 'Clicking on this icon will open a window that displays all URLs that you have connected to in the past (provided your browser permits storing in Local Storage).'
+				},
+
+				'login-options': {
+					target: '#credentials > .tabsHeader',
+					title: 'Login Options',
+					content: `You have several ways of authenticating with the server. The standard method is OpenID Connect (OIDC). If supported, you will see a button for each available OIDC provider here. Click on one to open its login form.`
+				},
+				'login-internal': {
+					target: '#credentials > .tabsHeader > button[title=Internal]',
+					title: 'User/Password',
+					content: `Some servers support basic authentication with a simple username-password combination. This method is discouraged and generally only used for internal development purposes (hence the name of this button). We suggest to use OpenID Connect if you can.`
```

Comment:
```suggestion
					content: `Some servers support "Basic authentication", which is usually only used for development and testing purposes. You just need to provide your username and password here, but usually you'd want to select one of the other options, if available.`
```

Reply:
The "You should choose something else if possible" is not included in the UI yet, so I would add a separate description, yes.

You usually made the explanations less technical, but in this instance you made it MORE technical with `"Basic authentication"`. On purpose?? I used "basic" as an adjective, not as part of the noun.

TBH I liked my version more. (Of course mentioning OpenID Connect needs to be removed if it's not mentioned in the other text.) Maybe I can combine the best of both.

Reply:
Yes, it was on purpose, as the description was pretty ambiguous. What does "basic authentication" actually mean? So I chose to make clear that this is the name. But indeed, it's still technical, so we may want to go one step further:

```suggestion
					content: `Some servers support an additional authentication method, which is best-suited and recommended for development and testing purposes only. You just need to provide your username and password here, but usually, you'd want to select one of the other options, if available.`
```

Reply:
You're right, "basic" didn't make as much sense as I thought at first. I like your suggestion. The only remark I have left: You only encourage OIDC, not explicitly "discourage" Basic.

Reply:
Hmm, isn't "recommended for development and testing purposes only" discouraging?

Reply:
The other way round would be:

> Some servers support an additional authentication method for development and testing purposes, which is discouraged for production use.

Reply:
I'll go with your last suggestion (not the quoted one).